nengo/tests/test_ensemble.py

```diff
@@ -388,13 +389,16 @@ def test_noise_copies_ok(Simulator, nl_nodirect, seed, plt, allclose):
 
     process = FilteredNoise(synapse=nengo.Alpha(1.0), dist=Choice([0.5]))
     with nengo.Network(seed=seed) as model:
-        inp, gain, bias = 1, 5, 2
-        model.config[nengo.Ensemble].neuron_type = nl_nodirect()
+        if "spikes" in nl_nodirect.state or RegularSpiking in nl_nodirect.__bases__:
```

Comment:
Could we just check if "voltage" is in `nl_nodirect.state`?

Reply:
There is no class-level `state` in `RegularSpiking`, it's an instance variable added during `__init__`, so we have to check the type of the class.